test/new-e2e/tests/installer/host/systemd.go

```diff
@@ -58,7 +58,7 @@ func eventMatches(log journaldLog, event SystemdEvent) bool {
 	if log.Unit != event.Unit {
 		return false
 	}
-	match, _ := regexp.MatchString(event.Pattern, log.Message)
```

Comment:
Case insensitiveness as older OSes (CentOS / AL2) use `failed` instead of `Failed`.